ztp/gitops-subscriptions/argocd/resource-hook-example/siteconfig/test-3node.yaml

```diff
@@ -0,0 +1,155 @@
+# test-node[123]-bmh-secret & assisted-deployment-pull-secret need to be created under same namespace test-3node
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: test-3node
+  labels:
+    name: test-3node
+---
+apiVersion: ran.openshift.io/v1
+kind: SiteConfig
+metadata:
+  name: "test-3node"
+  namespace: "test-3node"
+spec:
+  baseDomain: "example.com"
+  pullSecretRef:
+    name: "assisted-deployment-pull-secret"
+  clusterImageSetNameRef: "openshift-4.10"
+  sshPublicKey: "ssh-rsa AAAA..."
+  clusters:
+  - clusterName: "test-3node"
+    clusterType: "standard"
+    clusterProfile: "du"
+    numMasters: 3
+    clusterLabels:
+      group-du-sno: ""
+      common: true
+      sites : "test-3node"
+    clusterNetwork:
+      - cidr: 1001:db9::/48
+        hostPrefix: 64
+    machineNetwork:
+      - cidr: 2620:52:0:10e7::/64
+    serviceNetwork:
+      - 1001:db7::/112
+    additionalNTPSources:
+      - 2620:52:0:1310::1f6
+    nodes:
+      - hostName: "test-node1.example.com"
+        root: "master"
+        bmcAddress: "idrac-virtualmedia+https://[2620:52::10e7:f602:70ff:fee4:f4e2]/redfish/v1/Systems/System.Embedded.1"
+        bmcCredentialsName:
+          name: "test-node1-bmh-secret"
+        bootMACAddress: "0C:42:A1:8A:74:11"
+        bootMode: "UEFI"
+        rootDeviceHints:
+          hctl: '0:1:0'
+        cpuset: "0-1,52-53"
+        nodeNetwork:
+          interfaces:
+            - name: eno1
+              macAddress: "0C:42:A1:8A:74:11"
+          config:
+            interfaces:
+              - name: eno1
+                type: ethernet
+                state: up
+                macAddress: "0C:42:A1:8A:74:11"
+                ipv4:
+                  enabled: false
+                ipv6:
+                  enabled: true
+                  address:
+                  - ip: 2620:52::10e7:e42:a1ff:fe8a:901
+                    prefix-length: 64
+            dns-resolver:
+              config:
+                search:
+                - example.com
+                server:
+                - 2620:52:0:1310::1
+            routes:
+              config:
+              - destination: ::/0
+                next-hop-interface: eno1
+                next-hop-address: 2620:52:0:10e7::fc
+                table-id: 254
+      - hostName: "test-node2.example.com"
+        root: "master"
```

Comment:
Is this a field we can control? Prefer other terms rather than "master".

Reply:
Good catch - This was actually supposed to be `role: "master"` but because the default role is actually master, this typo was completely ignored by the tooling and the output was still correct.  Fixing.